dictionaries/wordpress.txt

```diff
@@ -0,0 +1,270 @@
+wordpress
+absint
+accessable
```

Comment:
In WP, I would think this is a typo of `accessible` rather than a new word

Reply:
Good catch. Looking through WordPress Core, the word is only found in `wp-includes/depreciated.php`, so it can be removed.